source/common/common/fine_grain_logger.cc

```diff
@@ -165,12 +165,28 @@ spdlog::logger* FineGrainLogContext::createLogger(const std::string& key)
   }
 
   new_logger->set_level(getLogLevel(key));
-  new_logger->set_pattern(Logger::Context::getFineGrainLogFormat());
+  setLogFormat(new_logger.get(), Logger::Context::getFineGrainLogFormat());
   new_logger->flush_on(level_enum::critical);
   fine_grain_log_map_->insert(std::make_pair(key, new_logger));
   return new_logger.get();
 }
 
+void FineGrainLogContext::setLogFormat(spdlog::logger* const logger,
```

Comment:
Can we have this just leverage shared code with the registry? https://github.com/envoyproxy/envoy/blob/6647c3f7d22bf8ab080f3b3d6ab4fb42cbb14629/source/common/common/logger.cc#L251-L266

That way they won't get out of sync.

Reply:
Done! Thank you.